GVFS/LibGit2Sharp.NativeBinaries.props

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
     <ItemGroup>
-        <None Condition="Exists('..\..\..\packages\LibGit2Sharp.NativeBinaries.1.0.165\runtimes\win7-x64\native\git2-1196807.dll')" Include="..\..\..\packages\LibGit2Sharp.NativeBinaries.1.0.165\runtimes\win7-x64\native\git2-1196807.dll">
+        <None Condition="Exists('..\..\..\packages\libgit2sharp.nativebinaries\1.0.165\runtimes\win7-x64\native\git2-1196807.dll')" Include="..\..\..\packages\libgit2sharp.nativebinaries\1.0.165\runtimes\win7-x64\native\git2-1196807.dll">

```

Comment:
Why was this change made?

Reply:
This is in the second commit's description in the PR:
> Fixes a bug that I introduced when porting our codebase to .NET Core. In RestorePacakges.bat, we only restored two of our .NET Standard projects. This results in these projects being restored into the user's home folder/.nuget. Since FastFetch was .NET Framework up until now, we never had issues with LibGit2Sharp not being present for Windows builds to consume. Once it was removed, we no longer had a place to consume the native .dll from. Now we properly restore that entire .sln and point it at the VFSForGit packages directory, not the default home folder location.